forc-plugins/forc-client/Cargo.toml

```
@@ -0,0 +1,34 @@
+[package]
+name = "forc-client"
+version = "0.19.2"
+edition = "2021"
+
+[dependencies]
+anyhow = "1"
+clap = { version = "3", features = ["derive", "env"] }
+forc-pkg = { path = "../../forc-pkg" }
```

This one's still missing a version, e.g.
```suggestion
forc-pkg = { version = "0.19.2", path = "../../forc-pkg" }
```